js/components/dateRange.js

```diff
@@ -36,7 +36,7 @@ Fliplet.FormBuilder.field('dateRange', {
     placeholder: {
       type: String,
       default: function() {
-        return '-- ' + T('widgets.form.dateRange.rangePlaceholder');
+        return '-- Select range';
```

Comment:
Can the Studio user customise this? If not, then we do need continue to use `T()` for this and show it to the user.